basket/news/utils.py

```diff
@@ -366,35 +360,9 @@ def get_user_data(
         'master': True if we found them in the master subscribers table
     }
     """
-    sfdc_enabled = True
-    try:
-        user = sfdc.get(
-            token=token,
-            email=email,
-            payee_id=payee_id,
-            amo_id=amo_id,
-            fxa_id=fxa_id,
-        )
-    except sfapi.SalesforceResourceNotFound:
-        return None
-    except requests.exceptions.RequestException as e:
-        raise NewsletterException(
-            str(e),
-            error_code=errors.BASKET_NETWORK_FAILURE,
-            status_code=400,
-        )
-    except sfapi.SalesforceAuthenticationFailed:
-        raise NewsletterException(
-            "Email service provider auth failure",
-            error_code=errors.BASKET_EMAIL_PROVIDER_AUTH_FAILURE,
-            status_code=500,
-        )
-    except SFDCDisabled:
-        sfdc_enabled = False
-        user = {}
+    user = {}
 
     if not user.get("email_id"):
```

Comment:
We no longer need this `if` statement, but there's enough churn as it is, so IMO it's fine to leave it. Might confuse someone in future though.

Reply:
I'm of the mind that if there's lots of churn, might as well get all the churn in at once so it's one commit of churn.